api/src/paths/project/{projectId}/survey/{surveyId}/critters/index.ts

```diff
@@ -163,6 +182,73 @@ POST.apiDoc = {
   }
 };
 
+PATCH.apiDoc = {
+  description:
+    'Creates a new critter in critterbase, and if successful, adds the a link to the critter_id under this survey.',
+  tags: ['critterbase'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  parameters: [
+    {
+      in: 'path',
+      name: 'surveyId',
+      schema: {
+        type: 'number'
+      },
+      required: true
+    }
+  ],
+  requestBody: {
+    description: 'Critterbase bulk creation request object',
+    content: {
+      'application/json': {
+        schema: {
+          type: 'object',
+          properties: {
+            create: {
+              type: 'object'
```

Comment:
Can this object params be declared? Same on `update:`

Reply:
Is it proper to have two separate objects for this as a PATCH endpoint? 
A similar problem has been approached in SIMS around updating, our solution is to implore `upsert` functions.  See `survey-service.ts` @ `upsertSurveyFundingSourceData()`
Where a single update object is given and each item is checked to be updated/added/removed

Reply:
Maybe so, but I think that would involve too significant of a change to critterbase for the scope of this ticket.

Reply:
https://stackoverflow.com/questions/28459418/use-of-put-vs-patch-methods-in-rest-api-real-life-scenarios

`PATCH` is for partial record updates, so the upsert pattern is a little more on-brand for a `PUT` request.